internal/class.h

```diff
@@ -14,6 +14,7 @@
 #include "internal/serial.h"    /* for rb_serial_t */
 #include "ruby/internal/stdbool.h"     /* for bool */
 #include "ruby/intern.h"        /* for rb_alloc_func_t */
+#include "constant.h"      /* for rb_const_entry_t */
```

Comment:
May as well align with the comments on the surrounding lines
```suggestion
#include "constant.h"           /* for rb_const_entry_t */
```